### PR TITLE
Add logstash collector FAT to OpenLiberty

### LINKED ISSUE #12379: Move com.ibm.ws.logstash.collector_fat to OpenLiberty
LogstashCollector-1.0 has been moved from WS-CD to OpenLiberty in #9291.  This issue is to track the move of `com.ibm.ws.logstash.collector_fat` from WS-CD to OpenLiberty.

### PR DESCRIPTION
**Pull request by Yushan-Lin**
Fixes #12379

### CODE REVIEW

**Comment by Yushan-Lin** [comment]
#build

**Comment by Yushan-Lin** [comment]
https://wasrtc.hursley.ibm.com:9443/jazz/web/projects/WS-CD#action=com.ibm.team.build.viewResult&id=_RWtw4KveEeqCjdhsjSafyw

**Comment by Yushan-Lin** [comment]
#run-libby-bot

**Comment by fmhwong** [comment]
Please check with Chuck Bridgham and see if we are still using git lfs to check in large files.  The zip file in this PR is 97MB.

**Comment by fmhwong** [comment]
Some files are using a mix of tab and spaces for indentation.  Can you convert them to all spaces?

File: dev/com.ibm.ws.logstash.collector_fat/publish/files/startLogstash.bat [hunk 1]
```diff
@@ -0,0 +1 @@
+lib\LibertyFATTestFiles\logstash-2.1.1\bin\logstash.bat -f lib\LibertyFATTestFiles\logstash-2.1.1\logstash.conf --verbose > lib\LibertyFATTestFiles\logstash_console.txt
```
Comment (fmhwong) [inline dev/com.ibm.ws.logstash.collector_fat/publish/files/startLogstash.bat:1]: This file is probably not used anymore as it is referring to an old version of Logstash.

File: dev/com.ibm.ws.logstash.collector_fat/.gitignore [hunk 2]
```diff
@@ -0,0 +1,3 @@
+dropins
+logstash-2.1.1
```
Comment (fmhwong) [inline dev/com.ibm.ws.logstash.collector_fat/.gitignore:2]: 2.1.1 is not used anymore.

**Comment by fmhwong** [comment]
What is in `dev/com.ibm.ws.logstash.collector_fat/.classpath.swp`?

**Comment by Yushan-Lin** [comment]
@fmhwong Chuck said they don't

**Comment by Yushan-Lin** [comment]
https://wasrtc.hursley.ibm.com:9443/jazz/web/projects/WS-CD#action=com.ibm.team.build.viewResult&id=_kb_N8K_WEeqCjdhsjSafyw

**Comment by Yushan-Lin** [comment]
https://wasrtc.hursley.ibm.com:9443/jazz/web/projects/WS-CD#action=com.ibm.team.build.viewResult&id=_DnFGALCdEeqCjdhsjSafyw

**Comment by Yushan-Lin** [comment]
https://wasrtc.hursley.ibm.com:9443/jazz/web/projects/WS-CD#action=com.ibm.team.build.viewResult&id=_LmrVoLWGEeqI3q2xeI2nKA

**Comment by Yushan-Lin** [comment]
https://wasrtc.hursley.ibm.com:9443/jazz/web/projects/WS-CD#action=com.ibm.team.build.viewResult&id=_x7x2ALWLEeqI3q2xeI2nKA

**Comment by Yushan-Lin** [comment]
https://wasrtc.hursley.ibm.com:9443/jazz/web/projects/WS-CD#action=com.ibm.team.build.viewResult&id=_CEnSsLoqEeqsDL_UN9umug

**Comment by Yushan-Lin** [comment]
https://wasrtc.hursley.ibm.com:9443/jazz/web/projects/WS-CD#action=com.ibm.team.build.viewResult&id=_LRY3ULsGEeqsDL_UN9umug&tab=com.ibm.team.build.web.ui.internal.editors.result.overview.OverviewPage

**Comment by Yushan-Lin** [comment]
https://wasrtc.hursley.ibm.com:9443/jazz/web/projects/WS-CD#action=com.ibm.team.build.viewResult&id=_ZJHBQL-JEeq9OZzIliLkpg&tab=com.ibm.team.build.web.ui.internal.editors.result.overview.OverviewPage

**Comment by Yushan-Lin** [comment]
#run-libby-bot

**Pull request merged**